src/amp-story-embed.js

```diff
@@ -88,33 +125,67 @@ export class AmpStoryEmbed {
   }
 
   /**
-   * @param {number} index
+   * @param {!Element} story
    * @private
    */
-  buildIframe_(index) {
-    const story = this.stories_[index];
-    this.iframeEl_ = this.doc_.createElement('iframe');
+  buildIframe_(story) {
+    const iframeEl = this.doc_.createElement('iframe');
     setStyle(
-      this.iframeEl_,
+      iframeEl,
       'backgroundImage',
       story.getAttribute('data-poster-portrait-src')
     );
-    this.iframeEl_.classList.add('story');
-    this.initializeLoadingListeners_();
-    this.rootEl_.appendChild(this.iframeEl_);
+    iframeEl.setAttribute(IFRAME_ID_ATTR_NAME, 'i' + this.iframeIdCounter_++);
```

Comment:
What's the plan for this in the future? It doesn't seem to be used at all in this PR
If we do need to keep it, could it be a className?

Reply:
I had it to keep track and reference the iframes using their id's, which is why I wanted to use id or an attribute. But now that I am saving the reference to the iframes this is no longer needed.